tests/report_tests/options_tests/options_tests.py

```diff
@@ -41,5 +41,5 @@ def test_plugopts_actually_set(self):
 
     def test_effective_options_logged_correctly(self):
         self.assertSosLogContains(
-            "effective options now: --batch --case-id 8675309 --plugopts kernel.with-timer=on,kernel.trace=yes --skip-plugins networking,logs"
+            "effective options now: --batch --case-id 8675309 --only-plugins host,kernel --plugopts kernel.with-timer=on,kernel.trace=yes"
```

Comment:
The `effective options` in fact do contain `--plugopts kernel.with-timer=on,kernel.trace=yes,networking.traceroute=yes`, even the option for disabled `networking` plugin. That is a tiny bug of resolving options that has no effect to the data collected (so imho not worth fixing it). Just this assert string is made in the way to work well even if the tiny bug would be fixed.